rust/worker/src/segment/record_segment.rs

```diff
@@ -579,6 +582,7 @@ impl SegmentFlusher for RecordSegmentFlusher {
     }
 }
 
+#[derive(Clone)]
```

Comment:
nit - is a use of this introduced somehwhere? couldn't find it but probably missed it

Reply:
Yes used https://github.com/chroma-core/chroma/pull/2397/files#diff-df526c8c584494e9936f747868fb1f418b2df6e9f042eb991847c3c471bc6b33R137